swap/swap.go

```diff
@@ -80,8 +85,41 @@ func NewParams() *Params {
 	}
 }
 
+// newLogger returns a new logger
+func newLogger(logpath string) log.Logger {
+	swapLogger := log.New("swaplog", "*")
+
+	defaultHandler := log.Root().GetHandler()
+	rfh, err := swapRotatingLogHandler(logpath)
```

Comment:
please name the variables consistently: `defaultHandler` and `rfh` are not congruent

Reply:
fixed